requirements.txt

```diff
@@ -3,7 +3,8 @@ numpy==1.22.3
 pandas==1.4.2
 requests==2.27.1
 scipy==1.7.3
-tensorflow==2.7.2
+tensorflow-macos==2.7.0
```

Comment:
is this required?

Reply:
Apologies this is done by mistake. Actually tensorflow-macos is for MAC M1, I was trying to install the deps. I will revert that.